src/__tests__/cat.test.ts

```diff
@@ -225,9 +314,208 @@ for (const format of ['symbolic', 'semantic'] as Array<'symbolic' | 'semantic'>)
+    const probsOutsideBounds = probs.filter((p, i) => xs[i] < -2 || xs[i] > 2);
+    probsOutsideBounds.forEach(prob => {
+      expect(prob).toBeCloseTo(0, 6);
+    });
+    
+    // Check that the total probability sums to 1
+    const totalProb = probs.reduce((sum, prob) => sum + prob, 0);
+    expect(totalProb).toBeCloseTo(1, 6);
+    
+    // Check that the range is correct (should use priorPar bounds, not minTheta/maxTheta)
+    expect(Math.min(...xs)).toBeCloseTo(-3, 6);
+    expect(Math.max(...xs)).toBeCloseTo(3, 6);
+  });
+    
+
+  it('should use default priorPar for uniform distribution', () => {
+    const cat = new Cat({ priorDist: 'unif' });
+    expect(cat.priorDist).toBe('unif');
+    expect(cat.priorPar).toEqual([-4, 4]);
+    expect(cat.prior.length).toBeGreaterThan(0);
+  });
+
+  it('should use fallback normal(0,1) when priorDist is invalid', () => {
+    // This test covers the fallback branch: normal(0, 1, minTheta, maxTheta)
+    // We need to test this by temporarily modifying the validation to allow invalid priorDist
+    const originalValidatePrior = Cat['validatePrior'];
+    
+    // Temporarily replace validatePrior to allow invalid priorDist
+    Cat['validatePrior'] = () => {
+      // No-op validation to allow invalid priorDist
+    };
+    
+    try {
+      const cat = new Cat({ 
+        minTheta: -2, 
+        maxTheta: 2,
+        priorDist: 'invalid' as any,
+        priorPar: [5, 10]
+      });
+      
+      // Check that the fallback was used (should be normal(0,1) over [-2,2])
+      const priorXValues = cat.prior.map(([x]: [number, number]) => x);
+      const priorYValues = cat.prior.map(([, y]: [number, number]) => y);
+      
+      // Check that the middle value is the largest (normal distribution should peak at the middle)
+      const middleIndex = Math.floor(priorYValues.length / 2);
+      const middleValue = priorYValues[middleIndex];
+      const maxValue = Math.max(...priorYValues);
+      expect(middleValue).toBeCloseTo(maxValue, 6);
+      
+      expect(Math.min(...priorXValues)).toBeCloseTo(-2, 6);
+      expect(Math.max(...priorXValues)).toBeCloseTo(2, 6);
+    } finally {
+      // Restore original validation method
+      Cat['validatePrior'] = originalValidatePrior;
+    }
+  });
+
+  it('uniform() outputs correct probabilities and boundaries', async () => {
+    const result = uniform(-2, 2, 0.5, -3, 3);
+    const probs = result.map(([, p]: [number, number]) => p);
+    const xs = result.map(([x]: [number, number]) => x);
+    // Probabilities sum to 1
+    expect(probs.reduce((a: number, b: number) => a + b, 0)).toBeCloseTo(1, 6);
+    // Boundaries have nonzero probability
+    expect(probs[xs.indexOf(-2)]).toBeGreaterThan(0);
+    expect(probs[xs.indexOf(2)]).toBeGreaterThan(0);
+    // Outside bounds are zero
+    expect(probs[xs.indexOf(-3)]).toBeCloseTo(0, 6);
+    expect(probs[xs.indexOf(3)]).toBeCloseTo(0, 6);
+  });
+
+});
```

Comment:
Can we move this test into `utils.test.ts` since the `uniform` function is defined in `utils.ts`?